### ISSUE TITLE
Windows 32 bit support

### ISSUE THREAD

**Issue by visr** [visr/GDAL.jl#24]
AppVeyor tests are failing on 32 bits:
```
LoadError: could not load symbol "GDALVersionInfo"
```

Looking at the 32 bit DLL is see it's called `_GDALVersionInfo@4` instead. This should not be a problem if we add the argument `stdcall` to `ccall` as explained [here](http://docs.julialang.org/en/release-0.4/manual/calling-c-and-fortran-code/#calling-convention). I guess a solution would be doing [this](https://github.com/JuliaLang/julia/blob/ced360b631feac9aa872730b4d8bb59dccc7f05a/base/libc.jl#L216-L220) everywhere it is needed, but that's not very pretty.

Alternatively, if we make use of this setting in GDAL's `nmake.opt`:
```
# If you don't want some entry points to have STDCALL conventions, 
# comment out the following and add -DCPL_DISABLE_STDCALL in OPTFLAGS.
# This option has no effect on 64-bit windows.
STDCALL=YES
```
I guess we could fix it by only changing the build. But then how do we do approach that?

Anyhow this is just a braindump of what I found trying to add support for 32 bit Windows, I'll stop here for now. But other ideas are welcome.

**Comment by yeesian** [comment]
@tkelman I'll be back in cambridge on the afternoon of the 22nd. Might I be able to catch you during juliacon to hand-hold me through the writing of a build script for GDAL (the way we did for LibGEOS)? I should take my own notes this time too.

**Comment by tkelman** [comment]
Most likely. I should be in Boston from the 20th to the 30th. You can try stdcall in ccall but it may not work all that well, ref https://github.com/JuliaLang/julia/issues/7375

**Comment by visr** [comment]
At some point I also tried going the OBS route, see
https://build.opensuse.org/project/show/home:visr
I'd have to dive in again to remember where I got stuck, but it may be of help.

What do you see as the main advantage of using openSUSE Build Service compared to Conda.jl? 
Ref #23

**Comment by tkelman** [comment]
OBS automatically rebuilds your package when any of its dependencies change, or when you make any modifications to the packaging script. Conda requires its own private python installation, which is pretty heavyweight if all you want is one C library. Windows binaries on Conda are generally built for compatibility with CPython, which means MSVC. OBS uses MinGW-w64, same as Julia is built with, so more likely to be compatible.

**Comment by visr** [comment]
Closed by #25
